apps/web/vibes/soul/components/blog-post-card/index.tsx

```diff
@@ -61,8 +63,8 @@ export const BlogPostCard = function BlogPostCard({
             })}
           </time>
         )}
-        {date && author && <span className="after:mx-2 after:content-['•']" />}
-        {author && author}
+        {date && author !== undefined && <span className="after:mx-2 after:content-['•']" />}
+        {author !== undefined && <span>author</span>}
```

Comment:
I think for these we want to check `date != null && author != null`. Also, seems like we're missing angled brackets around `author`, should be `author != null && <span>{author}</span>`